apps/betterangels-backend/accounts/models.py

```diff
@@ -79,11 +80,14 @@ class User(AbstractBaseUser, PermissionsMixin):
     servicerequestuserobjectpermission_set: models.QuerySet["ServiceRequestUserObjectPermission"]
 
     def __str__(self: "User") -> str:
-        return self.email
+        return f"{self.full_name} ({self.pk})"
```

Comment:
Hmm, why do we ahve the PK here?

Reply:
If this is for display wouldn't we want something like `{self.pk} - {self.full_name}`.  Let me know if there is a reasoning here.